ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/CommandCompletionsReader.scala

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.appendonlydao
```

Comment:
Nit-pick, so far we have separated words in the class paths with dots i.e.
`com.daml.platform.store.append.only.dao`

Reply:
valid point, I was also considering that, my arguments to keep it like this:
- simplicity in general, only one dir
- the old and new code is truely next to each other
- with access modifiers/current package references it would be harder to rename/refactor eventually
- this is ephemeral package: after cleanup we will end up with the same package structure

So therefore if you would agree, I propose to keep it simple on cost of less conforming to code-style principles

Reply:
I agree with Márton too keep it simple. How about removing the `dao` part, it's the append-only store and not the append-only-dao store :)

Reply:
@gerolf-da it's a bit different: not the store is getting forked, but rather the store.dao, so basically the original is the dao package, and the forked is the appendonlydao. But if you still think appendonly is enough, I have no strong opinion...anyway this is just a working title :)

Reply:
yeah, I actually made the comment 5 seconds before I started my own review. appendonlydao works for me. let's rather continue with the juicy bits and keep the bikeshedding for later :)